components/formats-gpl/src/loci/formats/in/InveonReader.java

```diff
@@ -110,7 +110,7 @@ public boolean isThisType(RandomAccessInputStream stream) throws IOException {
   {
     FormatTools.checkPlaneParameters(this, no, buf.length, x, y, w, h);
 
-    int planeSize = FormatTools.getPlaneSize(this);
+    long planeSize = (long) FormatTools.getPlaneSize(this);
```

Comment:
Given that FormatTools.getPlaneSize is returning an int and `sqrt(2^31) => 46340` we're likely to overflow for big histopathology images in general.  Should getPlaneSize (and the other helpers which use it) all not be switched to use `long`, saving the need for the cast here?  This will ensure we're not routinely overflowing for all readers using this and related helpers.